Assets/Scripts/ComponentSolvers/ComponentSolver.cs

```diff
@@ -76,6 +98,7 @@ public IEnumerator RespondToCommand(string userNickName, string message, IComman
         while (subcoroutine.MoveNext())
         {
             object currentValue = subcoroutine.Current;
+            int temp;
```

Comment:
I'm not a particularly big fan of variables called `temp`. Doesn't really indicate variable purpose. I can see that it's used for strike-count from the `award strikes` yielded string though.